index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Christian Bryant Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <div class="top"> <!--This div holds the Sparta logo, nav bar and banner -->
+        <img src="./images/spartalogo.png" alt="Sparta logo">
+
+        <nav>
+          <ul>
+            <li> <a href="#">About</a> </li>
+            <li> <a href="#">Contact</a></li>
+          </ul>
+        </nav>
+
+        <div class="banner"> <!--This div controls the banner title of the page-->
+
+          <h1>PORTFOLIO</h1>
+
+        </div>
+
+      </div>
+
+    </header>
+
+    <main>
+
+    <div class="container"> <!--This div controls the middle section of the page split into two parts-->
+
+      <div class="left"> <!--The left div holds information about me -->
+
+        <div class="left-top"> <!--The left-top div controls the heading and picture -->
+
+        <h1>Christian Bryant</h1>
+
+        <img src="./images/me.jpg" alt="" width="300" height="400">
+
+      </div>
+
+
```

Comment:
Some of this spacing between elements is inconsistent and pretty big - it can make it a bit difficult to get a sense of the structure of your HTML at a glance.